Admin/AutomationAdmin.php

```diff
@@ -76,7 +76,7 @@ public function configureViews(ViewCollection $viewCollection): void
     }
 
     /**
-     * {@inheritdoc}
+     * @return mixed[]
```

Comment:
We should contribute this back to `sulu/sulu` so we don't have this issue in every bundle.

If we are correct we would need to go with something like:

```
@return array<string, array<string, array<string, array<int, string>>>>
```

but think that would be to complex 🙈